webrtc/RTCConfiguration-iceCandidatePoolSize.html

```diff
@@ -1,35 +1,87 @@
 <!doctype html>
-<html>
-<head>
-<title>This test checks that RTCConfiguration.iceCandidatePoolSize is set</title>
+<meta charset="utf-8">
+<!--
+4.2.1 RTCConfiguration Dictionary
+
+  The RTCConfiguration defines a set of parameters to configure how the peer to peer communication established via RTCPeerConnection is established or re-established.
+
+  ...
+
+  iceCandidatePoolSize of type octet, defaulting to 0
+    Size of the prefetched ICE pool as defined in [JSEP] (section 3.5.4. and section 4.1.1.).
+-->
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-</head>
-<body>
 <script>
-function testValidPoolSize(size, expectedSize)
-{
-    test(function() {
-        var pc = new RTCPeerConnection({iceCandidatePoolSize: size})
-        assert_equals(pc.getConfiguration().iceCandidatePoolSize, expectedSize !== undefined ? expectedSize : size)
-    }, "Setting iceCandidatePoolSize to a valid value: " + size)
-}
-
-function testInvalidPoolSize(size)
-{
-    test(function() {
-        assert_throws(new TypeError(), () => new RTCPeerConnection({iceCandidatePoolSize: size}))
-    }, "Setting iceCandidatePoolSize to an invalid value: " + size)
-}
-
-testValidPoolSize(10)
-testValidPoolSize(0)
-testValidPoolSize(null, 0)
-testValidPoolSize(undefined, 0)
-
-testInvalidPoolSize(-1)
-testInvalidPoolSize(1000)
-testInvalidPoolSize("string")
+  function initializeWithValidPoolSize(iceCandidatePoolSize, expectedSize) {
```

Comment:
Nit: I find function names like `testInitializeWithValidPoolSize` to be more descriptive than `initializeWithValidPoolSize` on what the function is doing.